db/schema.rb

```diff
@@ -94,6 +95,11 @@
     t.index ["user_id", "last_used_at"], name: "index_device_user_id_last_used_at"
   end
 
+  create_table "disposable_domains", force: :cascade do |t|
```

Comment:
It didn't occur to me until now, but it would be nice to incorporate the word "email" in this name. The domains themselves aren't disposable, it's the fact that the domains offer a disposable email service.

```suggestion
  create_table "disposable_email_domains", force: :cascade do |t|
```